src/lib/docs/ContractKit/ContractKit.md

```diff
@@ -1,9 +1,58 @@
 ---
 title: ContractKit (Factory)
-description: change_me
+description: The Contract Kit is responsible for facilitating the creation of Contract instances through the load method.
 category: ContractKit
 published: true
 order: 1
 ---
 
-Creating a `ContractKit` instance and being able to `load` contract instances using it.
+# ContractKit Class
+
+The `ContractKit` class provided by the [Contract Kit](/docs/contract-kit) is a factory class that facilitates the creation of [Contract](/docs/contract-kit/contract) instances.
+
+## Creation
+
+When using the `SessionKit` within a web application, the developer will first establish an instance of the class with the required ContractKit arguments.
+
+A simple example would be as follows:
+
+```typescript
+import { ContractKit } from "@wharfkit/contract-kit"
+import { APIClient } from "@wharfkit/antelope"
+
+const contractKit = new ContractKit({
+    client: new APIClient("https://jungle3.greymass.com"),
+});
+```
+
+### Arguments
+
+The first parameter passed to the `ContractKit` is an object containing one required configuration.
```

Comment:
The latter half of this sentence is kinda confusing. "containing one required configuration" sounds robotic 😂

Reply:
🤖

Reply:
agreed lol, will fix 👍